### ISSUE TITLE
Fonts not loading

### ISSUE THREAD

**Issue by jetpacktuxedo** [AndrewBelt/WiTeX#7]
If I try to go directly to the new file link (through the CDN) I get an access denied

**Comment by AndrewBelt** [comment]
These URLs work for me. Anyone else having this problem?

```
http://cdn.rawgit.com/AndrewBelt/WiTeX/master/fonts/lmroman10-regular.woff
http://cdn.rawgit.com/AndrewBelt/WiTeX/master/fonts/lmroman10-bold.woff
http://cdn.rawgit.com/AndrewBelt/WiTeX/master/fonts/lmroman10-italic.woff
http://cdn.rawgit.com/AndrewBelt/WiTeX/master/fonts/lmroman10-bolditalic.woff
https://cdn.rawgit.com/AndrewBelt/WiTeX/master/fonts/lmroman10-regular.woff
https://cdn.rawgit.com/AndrewBelt/WiTeX/master/fonts/lmroman10-bold.woff
https://cdn.rawgit.com/AndrewBelt/WiTeX/master/fonts/lmroman10-italic.woff
https://cdn.rawgit.com/AndrewBelt/WiTeX/master/fonts/lmroman10-bolditalic.woff
```

`curl $(cat files.txt) > /dev/null`

**Comment by eigengrau** [comment]
The problem seems to be that the URLs in `style.css` don’t include the http scheme. When added, it works just fine.

**Comment by AndrewBelt** [comment]
What browser are you using? As far as I know, not including the scheme works on Firefox and Chrome (it defaults to the scheme of the main URL). Perhaps I should just use https for all of them.

**Comment by eigengrau** [comment]
I observed this on Firefox 37 & 38.

**Comment by AndrewBelt** [comment]
Fixed with db66dd957f8c2bc8b16b49554a6cc4e4504a5914. Sorry about the wait---I had to see this for myself. It only seems to be an issue on Windows, but using https always is a decent fix.
